k6/.envexample

```diff
@@ -10,3 +10,4 @@ DEFAULT_GRACEFUL_STOP=
 FILTER_TEST=
 DEBUG_MODE=
 SIGNED_TXS=
+ETH_BLOCK_NUMBER_CACHE_TTL_MS=
```

Comment:
nit: new line in next PR

Reply:
there's an intellij setting somewhere for this ...

editor/general/on save/ensure every saved file ends with a line break